### ISSUE TITLE
"main" is missing from bower.json

### ISSUE THREAD

**Issue by nathanpen** [rochal/jQuery-slimScroll#246]
In the bower.json file. a main entry point needs to be added. when using packages like 'wiredep' the main file is not found because it is not declared in this file

**Comment by anaskhl** [comment]
Yes exactly- It did not work with bower-installer because of this!

**Comment by pablolendsum** [comment]
I had to add the main entry as well, but this defeats the point of having bower-installer, since I'm not adding the bower-compontents folder to the solution, and everyone in my team would have to add this entry in their bower.json file of the bower-components folder.

Please fix this so the installation of the package adds this main entry in the bower.json file!!!

**Comment by rochal** [comment]
Hi all,

Please raise a pull request, happy to add this in. I don't work with bower so the information above doesn't tell me what exactly needs to change.
